### DISCUSSION TITLE
NPM Build failing all of the sudden due to PHPCS sniff violation

### DISCUSSION THREAD

**Discussion by munts** [flyntwp/flynt#466] [Q&A] [answered]
All of the sudden my NPM Build is failing and the error it gives is for a line of code that has worked fine on several builds until now.  Here is line 16 of the NavigationMain/functions.php: $data['menu'] = new Timber\Menu('navigation_main');

and below is the error message I am now getting.

FILE: /data/Components/NavigationMain/functions.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 16 | ERROR | [x] Parentheses must be used when instantiating a new
    |       |     class
    |       |     (PSR12.Classes.ClassInstantiation.MissingParentheses)

I suppose I could disable the PHPCS or add a rule, but I am stumped as to why I am getting this error when it shouldn't be happening.

Thoughts?

Thanks,
Scott

**Comment by timohubois** [comment, marked as answer]
Hi @munts, do you use the [current version](https://github.com/flyntwp/flynt/blob/cee63d8b425eed23ac5a0903f5f510d4790eebf1/Components/NavigationMain/functions.php) of the functions.php of this component?

Instead of 
`$data['menu'] = new Timber\Menu('navigation_main');`

should be:
`$data['menu'] = new Menu('navigation_main');`

And then you should also add: 
`use Timber\Menu;`

You can find the full source code here: 
[Components/NavigationMain/functions.php](https://github.com/flyntwp/flynt/blob/cee63d8b425eed23ac5a0903f5f510d4790eebf1/Components/NavigationMain/functions.php)
